app/models/concerns/system_like.rb

```diff
@@ -19,4 +19,24 @@ def compliant
     end
     result
   end
+
+  def total_rules(profile = nil)
```

Comment:
I'm not crazy about this name. It sounds like some sort of sum. It's also returning RuleResult results for only the last import. Maybe something like `last_scan_results`?